tests/unit/dom/focus-test.js

```diff
@@ -35,6 +35,27 @@ module('DOM Helper: focus', function (hooks) {
     document.getElementById('ember-testing').innerHTML = '';
   });
 
+  test('it executes registered focus hooks', async function (assert) {
+    assert.expect(3);
+
+    element = document.createElement('input');
+    insertElement(element);
+
+    let startHook = registerHook('focus', 'start', () => {
+      assert.step('focus:start');
+    });
+    let endHook = registerHook('focus', 'end', () => {
+      assert.step('focus:end');
+    });
+
+    await focus(element);
+
+    assert.verifySteps(['focus:start', 'focus:end']);
+
+    startHook.unregister();
+    endHook.unregister();
+  });
```

Comment:
Commenting here, but all of these tests need the same treatment:

```suggestion
  test('it executes registered focus hooks', async function (assert) {
    assert.expect(3);

    element = document.createElement('input');
    insertElement(element);
    
    let startHook = registerHook('focus', 'start', () => {
      assert.step('focus:start');
    });
    let endHook = registerHook('focus', 'end', () => {
      assert.step('focus:end');
    });

    try {
      await focus(element);

      assert.verifySteps(['focus:start', 'focus:end']);
    } finally {
      startHook.unregister();
      endHook.unregister();
    }
  });
```

This ensures that should a failure / error occur the rest of the tests aren't being broken...